troposphere/settings/local.py.j2

```diff
@@ -17,12 +17,15 @@ API_V2_ROOT = API_SERVER + "/api/v2"
 
 SITE_TITLE= "{{ SITE_TITLE}}"
 SITE_FOOTER= "{{ SITE_FOOTER }}"
+
 {% if THEME_NAME is defined and not THEME_NAME == "default" %}
 THEME_NAME="{{ THEME_NAME }}"
 {% else %}
-THEME_NAME="troposphere_theme"
+THEME_NAME= "troposphere_theme"
```

Comment:
Discussion on theming approaches w.r.t. CyVerse, JetStream and "Neutral/Troposphere" will be held at [ATMO-1159] (https://pods.iplantcollaborative.org/jira/browse/ATMO-1159)